apiserver/test/test_apiserver.py

```diff
@@ -0,0 +1,6 @@
+import hail as hl
+
+
+def test_count_range():
+    assert hl.backend_type() == 'ServiceBackend'
+    assert hl.utils.range_table(1000)._force_count() == 1000
```

Comment:
I like the service backend bit, but I think this is basically dominated by the Hail Python tests.  My intention was to run all the unit tests against the (since that is the future) once it was easy to parallelize the tests.